README.md

```diff
@@ -1,18 +1,18 @@
-# cri-containerd
+# cri
```

Comment:
The README.md may need some rework.

Explain the transition from cri-containerd to containerd.

The dependency/build section doesn't make much sense as well...

Let's only change the link... I'll rewrite this later...

Reply:
yes all docs need a detailed clean up... I just did a super fast clean up to get most of the low hanging fruit and to verify naming patterns for when to use containerd/cri, the cri plugin, cri by itself, CRI, etc...

Reply:
added to issue..